src/storage/redis_interface.py

```diff
@@ -20,8 +20,9 @@ def __init__(self, chunk_size=REDIS_MAX_VALUE_SIZE):
         redis_db = cfg.data_storage.redis_fact_db
         redis_host = cfg.data_storage.redis_host
         redis_port = cfg.data_storage.redis_port
+        redis_pw = cfg.data_storage.redis_pw or None  # empty str -> None
```

Comment:
This is not neccessary as `_parse_dict` from `src/config.py` does remove entrys that are only an empty string.
This in turn results to pydantic setting it to `None`.

Reply:
That is good to know. I will remove it then.